torch/csrc/distributed/rpc/process_group_agent.h

```diff
@@ -12,6 +12,11 @@ namespace torch {
 namespace distributed {
 namespace rpc {
 
+struct ProcessGroupRpcAgentOptions : public RpcAgentOptions {
+  ProcessGroupRpcAgentOptions() {}
```

Comment:
Default constructor should go, or defined as `= default`.

Reply:
Updating.